### ISSUE TITLE
.gitIgnore Needs updated

### ISSUE THREAD

**Issue by ccapasso01** [ppoffice/hexo-theme-hueman#249]
Please make sure these boxes are checked before submitting your issue. Thank you!

- [x] I have setuped and configurated the blog according to [Hexo official documentation](https://hexo.io/);
- [x] I have read the [Theme Wiki](https://github.com/ppoffice/hexo-theme-hueman/wiki) carefully and created my own configuration file(_config.yml);
- [x] I have looked up the [Issues](https://github.com/ppoffice/hexo-theme-hueman/issues) and found no duplicate issues.

Maybe I'm missing something but after following the documentation and renaming `_config.yml.example in the theme folder to _config.yml` the site would not render properly or load.  

After looking around I found that inside of .gitignore you have _config.yml.  Once removed, this solved the issue.  As such, I just want to confirm if .gitignore needs updated as well as your readme or if I am missing something?

**Comment by ppoffice** [comment]
@ccapasso01 The reason `_config.yml` is in `.gitignore` is to prevent I from uploading the `_config.yml` to the repository. This will avoid overwriting the existing `_config.yml` customized by the user every time he/she does a `git pull` or `git merge`. Of course, you can always fork the repository and remove that line from the ignore file if you want to sync your configurations.

**Comment by ccapasso01** [comment]
But that _config.yml file is in the theme dir and needed for the theme to specify its configuration.  Without it, the site doesn't load properly (unless I am mistaken).  When the user needs to push to the repo, doesn't that file need to be there?

**Comment by ppoffice** [comment]
@ccapasso01 Sometimes people just build their site locally and push the built files to their servers. If you do use some automatic building processes, you can uncomment the `_config.yml` from `.gitignore`. The point is to make sure the updates in this public repo don't change user's local `_config.yml`.

**Comment by ccapasso01** [comment]
Good points.  Only suggestion I would have would be to maybe put a note in your readme/documentation that if you are trying to push this repo as is (after modifying _config.yml) that you will need to update .gitignore.

For example, I followed your guide, renamed _config.yml.example  as instructed and when I pushed to GitHub/GitLab, my site didn't work.  For newer people (like myself), it would be helpful to know that .gitignore may need updated as well. 

Thank you for all your work on this theme by the way 👍
